recording.py

```diff
@@ -15,7 +15,8 @@ def main():
     params = {
         'trial_duration': 4,
         'trials_per_stim': 1,
-        'trial_gap': 1,
+        'trial_gap': 3,
```

Comment:
Because of the new protocol, `trial_gap` no longer represents the actual gap between trials but rather the amount of time that "get ready` is displayed, am I correct?

If so, please change all instances of `trial_gap` to `get_ready_duration` or something else if you prefer